sklearn/manifold/mds.py

```diff
@@ -429,3 +519,48 @@ def fit_transform(self, X, y=None, init=None):
             return_n_iter=True)
 
         return self.embedding_
+
+    def _fit_transform_ext(self, X):
+        self.fit(X)
+        return self.transform(X)
+
+    def center_similarities(self, D_aX, D_XX):
```

Comment:
I think this should be a private method, unless you have a very good reason otherwise.